### ISSUE TITLE
Issue using Import-AzSentinelAlertRule

### ISSUE THREAD

**Issue by garybushey** [wortell/AZSentinel#27]
Not sure if this is an error with this code or with the way the default queries are written in the Azure Sentinel GitHub repository.

I tried to use Import-AzureSentinelAlertRule going against the exchange_auditlogdisabled.yaml rule stored in the Detections/OfficeActivity folder in the Azure Sentinel GitHub repository (copied locally of course) and received the following error:

Import-AzSentinelAlertRule : Unable to initiate class with error: Exception setting "TriggerOperator": "Cannot convert value "gt" to type "TriggerOperator". Error: "Unable to match the identifier name gt to a valid enumerator name. Specify one of the following enumerator names and try again:
GreaterThan, FewerThan, EqualTo, NotEqualTo""
At line:1 char:1
+ Import-AzSentinelAlertRule -WorkspaceName "LA-Sentinel-Beta" -Setting ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
+ FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,Import-AzSentinelAlertRule

ConvertFrom-Json : Conversion from JSON failed with error: Unexpected character encountered while parsing value: R. Path '', line 0, position 0.
At C:\Users\garyb\OneDrive\Documents\PowerShell\Modules\AzSentinel\0.6.1\AzSentinel.psm1:790 char:52
+ ...               $errorResult = ($errorReturn | ConvertFrom-Json ).error                                             +                                                  ~~~~~~~~~~~~~~~~                                                     + CategoryInfo          : NotSpecified: (:) [ConvertFrom-Json], ArgumentException                                       + FullyQualifiedErrorId : System.ArgumentException,Microsoft.PowerShell.Commands.ConvertFromJsonCommand

Even after fixing that issue I then get another issue with the way the Timespan is stored.

**Comment by pkhabazi** [comment]
hi @garybushey , this is a known issue and it's caused because the yaml files use the value "gt" and it should be "GreaterThan". The value "gt" is not a supported value. please update the "TriggerOperator" value to "GreaterThan" and you are good to go.

**Comment by pkhabazi** [comment]
this issue will be fixed in the new version which will be release in the upcoming days

**Comment by pkhabazi** [comment]
hi @garybushey it took a little bit longer than expected but this issue should have been solved in version 0.6.2 in current **development branch**, is it possible for you to test this issue? the PS Module in ps gallery isn't updated yes so you need to import the module manually

**Comment by pkhabazi** [comment]
this is solved in version 0.6.2 which is currently available in the PowerShell gallery

**Comment by deepanshumarwah** [comment]
Hi @pkhabazi 
I am trying to import multiple rules with `Import-AzSentinelAlertRule -WorkspaceName #### -SettingsFile .\Rules.json`. However I am receiving this error : 
`Import-AzSentinelAlertRule: Unable to initiate class with error: You cannot call a method on a null-valued expression.`

Here is the json format for Rules.json

```
{
    "analytics": [
        {
            "displayName": "string",
            "description": "string",
            "severity": "High",
            "enabled": true,
            "query": "SecurityEvent | where EventID == \"4688\" | where CommandLine contains \"-noni -ep bypass $\"",
            "queryFrequency": "5H",
            "queryPeriod": "5H",
            "triggerOperator": "GreaterThan",
            "triggerThreshold": 5,
            "suppressionDuration": "6H",
            "suppressionEnabled": false,
            "tactics": []
        }
    ]
}
```
